accounting/rewind.go

```diff
@@ -99,7 +99,14 @@ func AccountAtRound(ctx context.Context, account models.Account, round uint64, d
 		MinRound: round + 1,
 		MaxRound: account.Round,
 	}
-	txns, r := db.Transactions(ctx, tf)
+	ctx2, cf := context.WithCancel(ctx)
+	defer cf()
```

Comment:
Isn't this a noop? The later defer will be called first.

Reply:
This was on Tsachi's suggestion. If db.Transactions were to panic it ensures we still cleanup.

Reply:
We don't have panics in indexer code, @tsachiherman .

Reply:
Even if we had, indexer would just crash it that case. So cleanup would be unnecessary.

Reply:
I believe the http server catches panics, so cleaning up resources would be useful.

Reply:
could you add a comment for the first `cf()` then?

Reply:
done